### ISSUE TITLE
我看train.py里调用modules.py文件里的两个约束损失时传递了三个参数，但是参数未定义

### ISSUE THREAD

**Issue by huxiaohuxiaohuhu** [lsq960124/StyleBERT#3]
loss += hp.alpha * get_cmd_loss(diff_loss, speech_info, text_info)

loss += hp.beta * get_diff_loss(diff_loss, speech_info, text_info)

调用这两个方法时，对于语境化特征和差异化特征没有定义。
可以告诉我应该怎么定义speech_info和text_info吗。
